spec/requests/references/imports_controller_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe References::ImportsController do
+  subject(:rendered) { create_user && make_request && response }
```

Comment:
you can create_user inside the before block

Reply:
create_user is used in subject for both tests, so it has to be outside of describes